tests/tests_markdown_parser.py

```diff
@@ -0,0 +1,135 @@
+        """Title conversion shouldn't work
+        """
+        markdown = "# title"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p># title</p>\n"
+
+        assert result == expected_result
+
+    def test_parse_urls(self):
+        """Literal URLs auto-link https://foo.bar
+        """
+        markdown = "https://toto.space"
+        result = parse_markdown_description(markdown)
+        expected_result = (
+            '<p><a href="https://toto.space">https://toto.space</a></p>\n'
+        )
+
+        assert result == expected_result
+
+    def test_parse_urls_title(self):
+        """URLs with title [title for the link](https://foo.bar)
+        """
+        markdown = "[toto](https://toto.space)"
+        result = parse_markdown_description(markdown)
+        expected_result = '<p><a href="https://toto.space">toto</a></p>\n'
+
+        assert result == expected_result
+
+    def test_parse_italics(self):
+        """Italics (_foo_)
+        """
+        markdown = "_text_"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p><em>text</em></p>\n"
+
+        assert result == expected_result
+
+    def test_parse_bold(self):
+        """Bold (**foo**)
+        """
+        markdown = "**text**"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p><strong>text</strong></p>\n"
+
+        assert result == expected_result
+
+    def test_parse_paragraph_merging(self):
+        """Paragraph merging (consecutive lines are joined)
+        """
+        markdown = "this is\n a paragraph"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p>this is\n a paragraph</p>\n"
+
+        assert result == expected_result
+
+    def test_parse_paragraph(self):
+        """Paragraphs
+        """
+        markdown = "paragraph 1\n\n paragraph 2"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p>paragraph 1</p>\n<p>paragraph 2</p>\n"
+
+        assert result == expected_result
+
+    def test_parse_text(self):
+        """Text conversion works
+        """
+        markdown = "text"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p>text</p>\n"
+
+        assert result == expected_result
+
+    def test_parse_code_block(self):
+        """Code (text blocks inside ` or ``` pairs)
+        """
+        markdown = "```code block```"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p><code>code block</code></p>\n"
+
+        assert result == expected_result
+
+    def test_parse_code_line(self):
+        """Code (text blocks inside ` or ``` pairs)
+        """
+        markdown = "`code line`"
+        result = parse_markdown_description(markdown)
+        expected_result = "<p><code>code line</code></p>\n"
+
+        assert result == expected_result
+
+    def test_parse_list(self):
+        """Lists (* Foo)
```

Comment:
Can we extend this test to cover our custom lists as well?

Reply:
test added